src/me/ryanhamshire/GriefPrevention/command/CommandBuyClaimBlocks.java

```diff
@@ -0,0 +1,69 @@
+package me.ryanhamshire.GriefPrevention.command;
+
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.spec.CommandExecutor;
+
+public class CommandBuyClaimBlocks implements CommandExecutor {
+
+    @Override
+    public CommandResult execute(CommandSource src, CommandContext ctx) {
+        // TODO: Implement economy
```

Comment:
@phroa For when Economy is implemented, nobody has to go and make the whole command executor class - also this commented out code was already there, I just moved it to a separate class.